code/modules/mob/living/carbon/life.dm

```diff
@@ -27,6 +27,7 @@
 		. = ..()
 
 	if(stat == DEAD)
+		handle_death()
```

Comment:
Shouldn't this be handled by the liver?

Reply:
I was sort of stuck in the paradox of can a liver process if it's part of someone who is dead? I've moved it across to liver in the incoming push.